src/components/Chat/Chat.styled.js

```diff
@@ -6,11 +6,23 @@ export const ChatWrap = styled(Box)`
   display: flex;
   flex-direction: column;
   justify-content: space-between;
-  width: 620px;
-  height: 761px;
-  padding: 8px 8px 16px;
+  width: 100%;
+  max-width: 342px;
+  height: 707px;
```

Comment:
it is possible not to set the height, if users have different height displays, then there will be scrolling, if need height maybe use % or vh